src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/GetSemanticTokens_Razor_NestedTextDirectives.semantic.txt

```diff
@@ -1,20 +1,20 @@
 //line,characterPos,length,tokenType,modifier
-0 0 1 73 0 //razorTransition@
-0 1 9 76 0 //razorDirectivefunctions
```

Comment:
Why'd these all change?

Reply:
I had previously manually edited them to include the text that was being colorized at the end to help me think while working on a bug. I committed that, but when I re-ran the generator for these files it overwrote it. I figured that it was going to get overwritten every time and I was better off not re-adding the text unless I was going to add that to the generator.